website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
 
-Developers are also able to express an Indexer preference to the gateway, for example preferring Indexers whose query response is faster, or whose data is most up to date. These controls are set in Subgraph Studio.
+### Subgraph Lifecycle
 
-### Updating Subgraphs
+Here is a general view of a subgraph’s cycle:
 
-After a time a subgraph developer may want to update their subgraph, perhaps fixing a bug or adding new functionality. The subgraph developer may deploy new version(s) of their subgraph to Subgraph Studio for rate-limited development and testing.
+![Subgraph Lifecycle](/img/subgraph-lifecycle.png)
 
-Once the Subgraph Developer is ready to update, they can initiate a transaction to point their subgraph at the new version. Updating the subgraph migrates any signal to the new version (assuming the user who applied the signal selected "auto-migrate"), which also incurs a migration tax. This signal migration should prompt Indexers to start indexing the new version of the subgraph, so it should soon become available for querying.
+### Subgraph Development
 
-### Deprecating Subgraphs
+Subgraph development starts with local development and staging.
 
-At some point a developer may decide that they no longer need a published subgraph. At that point they may deprecate the subgraph, which returns any signalled GRT to the Curators.
+- You can use the same local setup whether you're building for The Graph Network or a local Graph Node.
+  - You can leverage `graph-cli` and `graph-ts` to build your subgraph.
+- You are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of your subgraphs.
 
-### Diverse Developer Roles
+> Only subgraphs on [supported networks](/developing/supported-networks/) will earn indexing rewards. Subgraphs that fetch data from IPFS are also **not** eligible.
 
-Some developers will engage with the full subgraph lifecycle on the network, publishing, querying and iterating on their own subgraphs. Some may be focused on subgraph development, building open APIs which others can build on. Some may be application focused, querying subgraphs deployed by others.
+## Deploy to Subgraph Studio
 
-### Developers and Network Economics
+Once defined, a subgraph can be built and deployed to [Subgraph Studio](/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
 
-Developers are a key economic actor in the network, locking up GRT in order to encourage indexing, and crucially querying subgraphs, which is the network's primary value exchange. Subgraph developers also burn GRT whenever a subgraph is updated.
+- Verify that your subgraph doesn't have any indexing errors and works as expected.
+- Use its staging environment to index the deployed subgraph and make it available for review.
+
+Learn how to [deploy subgraphs](/deploying-a-subgraph-to-studio/).
```

Comment:
We already link to that just above, I would probably just keep one of the links.